requirements.txt

```diff
@@ -1,31 +1,27 @@
-gunicorn
-
-Django>=3.2,<3.3
-django-webpack-loader==1.5.0
-mysqlclient
-django-mysql
-
-django-db-file-storage # Support for storage in the database
-Pillow # Needed for images
-django-tinymce # Rich text editor
-django-csp # For Content Security Policy
+gunicorn==20.1.0
+mysqlclient==2.1.1
+
+# Django and related
+Django==3.2.14
+django-csp==3.7 # For Content Security Policy
+django-db-file-storage==0.5.5 # Support for storage in the database
+django-mysql==4.7.0
+django-tinymce==3.4.0 # Rich text editor
+django-webpack-loader==1.6.0
+whitenoise==5.3.0 # For serving static files
 
 # DRF
-djangorestframework
-markdown       # Markdown support for the browsable API.
-django-filter  # Filtering support
+djangorestframework==3.13.1
+django-filter==22.1 # Filtering support
 drf-spectacular==0.22.1
+markdown==3.4.1 # Markdown support for the browsable API
 
-#LTI 
-PyLTI1p3==1.9.1
-
-# Not in pypi https://github.com/Harvard-University-iCommons/django-canvas-oauth
-https://github.com/Harvard-University-iCommons/django-canvas-oauth/archive/v1.1.0.tar.gz
-
-# For serving static files
-whitenoise==5.3.0
```

Comment:
Looks like there is a major release for `whitenoise`. Seems like it'd be low-risk, since we don't actually use the API very thoroughly and the Django version is supported. Thoughts? https://whitenoise.evans.io/en/stable/changelog.html

Reply:
I don't see anything in the changelog that should break, just different versions supported. Seems fine if it works. You'll know immediately if it doesn't, though I *think* it's only used in the Openshift build.

Reply:
Done, we'll have to test once it's built in OpenShift.